blockchain/test/test.go

```diff
@@ -0,0 +1,54 @@
+package test
```

Comment:
This file should be moved to the testutils package.

Reply:
Review task has been added.